### PR TITLE
fix slice tag bug AND add chinese name

### PR DESCRIPTION
**Pull request by qiangmzsx**
# fix bug
```
AdCode[]string `json:"ad_code" xorm:"ad_code" faker:"len=slice_len=5,len=10"` 
```
return：
```
{AdCode:[rMZLe XgkhW IZHBZ qEaWa nwrKJ]}
```

but
```
AdCode[]string `json:"ad_code" xorm:"ad_code" faker:"len=10,slice_len=5"`
```
return:
```
{AdCode:[RotjXUZmnq MTbKBCQhJC PedUsSqIwv xTnlUgWajY PBENmHwGfc]}
```
# features
- add chinese_first_name
- add chinese_last_name
- add chinese_name

### CODE REVIEW

**Comment by qiangmzsx** [comment]
I don't see the previously committed code being merged into master, is there anything I need to do?

**Comment by bxcodec** [comment]
Sorry, there's a linter issue. Not sure why, it's passed before merged to master, but it failed now.

**Comment by bxcodec** [comment]
Can you help to fix the linter?

**Comment by qiangmzsx** [comment]
I will deal with it as soon as possible and submit the code

**Comment by qiangmzsx** [comment]
Through testing I found some problems with the code I wrote before and fixed them. Now it should be ready for merging, thanks!

File: faker.go [hunk 1]
```diff
@@ -632,9 +652,31 @@ func decodeTags(typ reflect.Type, i int) structTag {
...
+		}
+	}
+	// custom,keep,unique
+	if len(res) < 1 {
+		if !keepOriginal && !uni {
+			res = append(res, tags...)
+		}
 	}
 
+	// fmt.Println("++TAGS:", tags, res)
```
Comment (bxcodec) [inline faker.go:679]: can remove this?

File: faker_test.go [hunk 2]
```diff
@@ -1954,6 +2020,7 @@ func TestOneOfTag__BadInputsForInts(t *testing.T) {
 		err := FakeData(&a)
 		if err == nil {
 			t.Fatal("expected error, but got no error")
+			return
```
Comment (bxcodec) [inline faker_test.go:2023]: why need to return? Is there any issue?

**Review by bxcodec — COMMENTED** [review]
Hey @qiangmzsx 

So sorry for the late reply, I was so busy in the past few months, just had time to review your PR now. 

I have a few comments, please re-review when you have time, thnx

**Comment by qiangmzsx** [comment]
Thanks for the review. I have made the changes and would appreciate your review again.

**Comment by bxcodec** [comment]
Hey @qiangmzsx  I think it's conflict, can you rebase your code from the master, and apply your changes?

**Comment by qiangmzsx** [comment]
I'm very sorry! This was my problem, caused by not doing a pull of the latest code for a long time, and I have fixed it. Request another review.

File: faker.go [hunk 3]
```diff
@@ -134,6 +137,15 @@ const (
 	//hyphen = "-"
 )
 
+// PriorityTags define the priority order of the tag
+var PriorityTags = []string{ID, HyphenatedID, EmailTag, MacAddressTag, DomainNameTag, UserNameTag, URLTag, IPV4Tag,
```
Comment (bxcodec) [inline faker.go:141]: remind me again, what was this priority tags used for?

File: faker.go [hunk 4]
```diff
@@ -653,7 +673,28 @@ func decodeTags(typ reflect.Type, i int) structTag {
 			uni = true
 			continue
 		}
-		res = append(res, tag)
+		// res = append(res, tag)
```
Comment (bxcodec) [inline faker.go:676]: please remove if not used anymore

**Comment by bxcodec** [comment]
Hey @qiangmzsx 
reviewed, a minor change, also I have a question. Please address when you're free thank you 🙇🏻

**Comment by qiangmzsx** [comment]
> 

I'm online today, so feel free to communicate with me if you need anything.

**Comment by qiangmzsx** [comment]
My email address is qiangmzsx@hotmail.com 。

**Review by bxcodec — APPROVED** [review]

**Pull request merged**